shell/browser/api/electron_api_web_contents_view.cc

```diff
@@ -78,38 +57,78 @@ WebContentsView::~WebContentsView() {
   }
 }
 
+gin::Handle<WebContents> WebContentsView::GetWebContents(v8::Isolate* isolate) {
+  return gin::CreateHandle(isolate, api_web_contents_);
+}
+
 void WebContentsView::WebContentsDestroyed() {
   api_web_contents_ = nullptr;
   web_contents_.Reset();
 }
 
+// static
+gin::Handle<WebContentsView> WebContentsView::Create(
+    v8::Isolate* isolate,
+    const gin_helper::Dictionary& web_preferences) {
+  v8::Local<v8::Context> context = isolate->GetCurrentContext();
+  v8::Local<v8::Value> arg = gin::ConvertToV8(isolate, web_preferences);
+  v8::Local<v8::Object> obj;
+  if (GetConstructor(isolate)->NewInstance(context, 1, &arg).ToLocal(&obj)) {
+    gin::Handle<WebContentsView> web_contents_view;
+    if (gin::ConvertFromV8(isolate, obj, &web_contents_view))
+      return web_contents_view;
+  }
+  return gin::Handle<WebContentsView>();
+}
+
+// static
+v8::Local<v8::Function> WebContentsView::GetConstructor(v8::Isolate* isolate) {
```

Comment:
Is it possible to make it a `gin_helper::Constructible`, would match the style of upcoming classes or is it better to handle it in a separate PR ?

Reply:
`Constructible` uses `FillObjectTemplate` instead of `BuildPrototype` to fill the prototype, and it requires declaring a static `kWrapperInfo` member, I would like to use it after converting all `View` classes to `gin::Wrappable`, otherwise the classes would be in mixed styles.